pkg/network/java/hotspot.go

```diff
@@ -143,14 +148,43 @@ func (h *Hotspot) copyAgent(agent string, uid int, gid int) (dstPath string, cle
 	}, nil
 }
 
+func (h *Hotspot) dialunix(raddr *net.UnixAddr, withCredential bool) (*net.UnixConn, error) {
+	// Hotspot reject connection if credential by checking uid/gid of the connect() SOL_SOCKET/SO_PEERCRED
```

Comment:
I didn't understand the sentence, perhaps couple of words are missing around `if credentials by...`

Reply:
updated